packages/myst-to-typst/tests/cite.yml

```diff
@@ -9,7 +9,7 @@ cases:
             - type: cite
               label: cockett2015
     typst: |-
-      @cockett2015
+      #cite(<cockett2015>)
```

Comment:
Can we only do this if necessary? i.e. leave it as a normal label for the most part and only change if we are overriding the default or something?

Reply:
skipping for now!